perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -4335,15 +4335,15 @@ public void convertAttributeValuesToUnique(PerunSession session, AttributeDefini
 								break;
 						}
 						int c = counter.addAndGet(1);
-						if(c%1000==0) log.info(""+c+" values of "+attrDef.getName()+" were converted");
+						if(c%1000==0) log.debug(""+c+" values of "+attrDef.getName()+" were converted");
```

Comment:
Do not use string concatenation for constructing log messages, use message formatting with {} as placeholders and variable arguments for values instead. Message formatting is more efficient if the active loglevel is less detailed than the level of this statement, because no new objects need to be created in that case.